objects/_objects.layout.scss

```diff
@@ -211,6 +211,10 @@ $inuit-use-markup-fix: false !default;
     display: flex;
   }
 
+  &.o-layout--center {
+    justify-content: center;
+  }
```

Comment:
That said, I find `o-layout--stretch` not very convincing here now as stretch suggests `align-items/content: stretch`. @inuitcss/core any thoughts?